includes/admin/upgrades/v3/class-data-migrator.php

```diff
@@ -534,25 +534,31 @@ public static function orders( $data = null ) {
 			$modified_time     = new \DateTime( $data->post_modified );
 			$modified_time_gmt = new \DateTime( $data->post_modified_gmt );
 
-			$diff = $modified_time_gmt->diff( $modified_time );
+			if ( $modified_time != $modified_time_gmt ) {
```

Comment:
@robincornett because I know you'll notice: this is intentionally not a strict check. We're comparing two `DateTime` objects, and a strict check won't actually work. But lose one does.